pixeltable/exprs/json_mapper.py

```diff
@@ -29,12 +33,18 @@ def __init__(self, src_expr: Expr, target_expr: Expr):
 
         from .object_ref import ObjectRef
 
-        scope_anchor = ObjectRef(self.target_expr_scope, self)
-        self.components = [src_expr, target_expr, scope_anchor]
+        self.components = [src_expr, target_expr]
         self.parent_mapper: Optional[JsonMapper] = None
```

Comment:
Can you add class-level type decls? I've been retrofitting them whenever I touch a class.

Reply:
Done.